sdk/textanalytics/Azure.AI.TextAnalytics/src/EntityCategory.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Azure.Core;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Gets the entity category inferred by the text analytics service's named entity recognition model.
+    /// The list of available categories is described at <see href="https://docs.microsoft.com/en-us/azure/cognitive-services/Text-Analytics/named-entity-types"/>.
+    /// </summary>
+    [JsonConverter(typeof(EntityCategoryJsonConverter))]
+    public readonly struct EntityCategory : IEquatable<EntityCategory>
```

Comment:
Will you have a separate struct for the `PiiEntity` categories and subcategories?

Reply:
wasn't thinking on creating a different one, just later create the PiiEntity type.
why? You?

Reply:
I guess what I'm really trying to point out is that the PII categories/subcategories from this list are not included in the struct: https://docs.microsoft.com/en-us/azure/cognitive-services/Text-Analytics/named-entity-types?tabs=personal

Reply:
yeah. didn't know we didn't have them all. 
Issue #9740 for it